### DISCUSSION TITLE
Using gradient checkpointing while fine-tuning XTTS V.2

### DISCUSSION THREAD

**Discussion by Aml-Hassan-Abd-El-hamid** [coqui-ai/TTS#3993] [General Q&A]
How can I specify the usage of gradient checkpointing to be true while fine-tuning XTTS V.2 -GPT 2- so I can use a larger batch size
